src/js/components/Anchor/index.js

```diff
@@ -1 +0,0 @@
-export { Anchor } from './Anchor';
```

Comment:
this is where i propose to add the HOC

export const Anchor = (props) => {
  if dev -> warn

  return <Link {...props} />
}